content/configuration/external-configs.md

```diff
@@ -4,10 +4,36 @@ sort: 3
 contributors:
   - sokra
   - skipjack
+  - tarang9211
 ---
 
-...
+Webpack lets you define you configuration files in any language. The list of supported file extensions can be found at the [node-interpret](https://www.npmjs.com/package/interpret) package. Webpack will still interpret your file as pure JS!
```

Comment:
According to GitHub, the package is now in a repository called js-interpret: https://github.com/js-cli/js-interpret